discord/emoji.py

```diff
@@ -65,7 +66,14 @@ class PartialEmoji(namedtuple('PartialEmoji', 'animated name id')):
         The ID of the custom emoji, if applicable.
     """
 
-    __slots__ = ()
+    __slots__ = ('animated', 'name', 'id')
+
+    def __init__(self, state, *, animated, name, id=None):
```

Comment:
By changing this constructor you have to ensure that all instantiations are also updated to meet the changes.

You missed one in `discord/ext/commands/converter.py`.